public/maint.html

```diff
@@ -30,51 +25,110 @@
       background-color: #151414;
       margin:0;
       padding:0;
-      min-height: 100%;
+      height: 100vh;
+      display: flex;
+      flex-direction: column;
     }
 
-    div.col {
-      margin: 0 auto;
-      padding-bottom: 20px;
+    div#image_container {
+      width: 100%;
+      height: calc(100vh - 200px);
+    }
+
+    div#image_container.narrow {
+      height: auto;
+    }
+
+    img#image {
       height: 100%;
+      display: block;
+      margin: 0 auto;
       max-width: 960px;
-      background: #353535;
     }
 
-    div.image {
-      max-width: 960px;
-      display: block;
-      background: #353535 top center / contain no-repeat url(/images/502.png);
-      padding-bottom: 92%;
+    img#image.narrow {
+      height: auto;
+      width: 100%;
+    }
+
+    div#message {
+      width: 100%;
+      height: 200px;  
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
     }
 
-    h1,h2 {
+    div#message.narrow {
+      height: auto;
+      padding-top: 30px;
+      padding-bottom: 30px;
+    }
+
+    h1, h2 {
       text-align:center;
       font-family: "Alright-v2-Normal";
       padding: 0 6%;
+      margin-top: 0;
+      margin-bottom: 0;
     }
 
     h1 {
       color: #E6E8E8;
-      margin-top: 0;
-      margin-bottom: 0;
     }
 
     h2 {
-      margin-top: 0;
-      margin-bottom: 0;
       color: #949797;
     }
+
+    div#message.narrow h1 {
+      font-size: 24px;
+    }
+
+    div#message.narrow h2 {
+      font-size: 18px;
+    }
+
   </style>
 </head>
 
-<body>
-
-  <div class="col">
-    <div class="image"> </div>
+<body>  
+  <div id="image_container">
+    <img id="image" src="/images/502.png" alt="">
+  </div>
+  <div id="message">
     <h1>alonetone is down for maintenance</h1>
     <h2>We probably took the site down on purpose! It should be back up shortly...</h2>
   </div>
+<script>
+
+  var viewportResized = function () {
```

Comment:
Thanks! We should probably stay consistent with the codebase and use `const` for variables that won't get reassigned and `let` for variables that get reassigned, otherwise perfecto, thanks for cooking this up!